### PR TITLE
feat: Add EV charger and car parking agents

### PR DESCRIPTION
**Pull request by coderlktripathi**
This PR introduces two new agents, 'ev_charger' and 'geoapi_car_parking', to enhance mobility services. These agents enable users to find available EV chargers and parking spaces in a specified region using latitude, longitude, and radius parameters.

The 'ev_charger' agent integrates with the OpenChargeMap API to provide real-time data on EV charger availability. On the other hand, the 'geoapi_car_parking' agent leverages the Geoapify API to offer comprehensive information about available parking spaces.

These new agents aim to improve the overall mobility experience, making it easier for users to locate charging stations for their electric vehicles and find suitable parking spots in their vicinity.

### CODE REVIEW

**Comment by GataKamsky** [comment]
@lrahmani / @jrriehl can one of you please review and approve this?

File: mobility-integrations/src/main.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from agents.ev_charger.ev_charger import agent as ev_charger_agent
+from agents.geopy_car_parking import agent as geopy_car_parking_agent
```
Comment (lrahmani) [inline mobility-integrations/src/main.py:2]:
```suggestion
from agents.geopy_car_parking.geopy_car_parking import agent as geopy_car_parking_agent
```

Comment (lrahmani) [reply mobility-integrations/src/main.py:2]: `agent` cannot be imported from `agents.geopy_car_parking`.

**Review by lrahmani — CHANGES_REQUESTED** [review]
Looks Good To Me, although the use case is not complete as there are no consumer agents and providers only suggest options. Please check inline comments for more details.

**Comment by coderlktripathi** [comment]
@lrahmani fixed import and Readme

**Review by lrahmani — APPROVED** [review]

**Pull request merged**